packages/lab/src/avatar/Avatar.css

```diff
@@ -1,66 +1,37 @@
 /* Default variables applied to the root element */
 .saltAvatar {
-  --avatar-background: var(--saltAvatar-background, var(--salt-accent-background));
   --avatar-foreground: var(--saltAvatar-foreground, var(--salt-accent-foreground));
+  --avatar-size-multiplier: var(--saltAvatar-size-multiplier, 1);
+  --avatar-base-size: var(--salt-size-base, 12px);
+  --avatar-base-fontSize: var(--saltAvatar-fontSize, var(--salt-accent-fontSize));
+  --avatar-container-size: calc(var(--avatar-base-size) * var(--avatar-size-multiplier));
+  --avatar-fontSize: calc(var(--avatar-base-fontSize) * var(--avatar-size-multiplier));
+  /* Icon styling */
+  --saltIcon-color: var(--avatar-foreground);
+  --salt-size-icon-base: calc(var(--avatar-container-size) / 2);
 }
 
 /* Style applied to the root element */
 .saltAvatar {
-  background: var(--avatar-background);
+  line-height: var(--salt-accent-lineHeight);
```

Comment:
if we expose --saltAvatar-fontSize, we should also expose --saltAvatar-lineHeight